### ISSUE TITLE
Problems with broken sentences in the bilingual subtitles generated

### ISSUE THREAD

**Issue by junruizh2021** [lewangdev/autotranslate#2]
Hi @lewangdev, thanks so much for your script. Sometimes I encounter problems with broken sentences in the bilingual subtitles generated during the process, resulting in **some source language subtitles missing translated subtitles**. 
Have you ever encountered this situation?
![image](https://github.com/lewangdev/autotranslate/assets/91523811/083bd820-bd0e-4b21-b585-504767398ce8)

Supplementing a bit, I am using a third-party API, not OpenAI API.

**Comment by lewangdev** [comment]
Yes, I have also seen this issue. Currently, the translation still requires manual intervention.

This problem is caused by the GPT-3.5. The prompt I am using currently unable to execute as expected with 100% accuracy. In my script, I set the default temperature to 0, and it will always get the same answer after retry multiple times.  

Some approaches I thought to sovle this problem:

1. set temperature to 0.7~1
1. A better prompt
2. Use GPT-4
3. Human involving

**Comment by lewangdev** [comment]
I will attempt to resolve this issue using OpenAI Function-Calling.
